scripts/LaunchOnCondor.py

```diff
@@ -372,7 +372,7 @@ def AddJobToCmdFile():
         if(not os.path.isabs(absoluteShellPath)): absoluteShellPath= os.getcwd() + "/"+absoluteShellPath
         Jobs_List.extend([absoluteShellPath])
     elif subTool=='slurm':
-        cmd_file.write('sbatch --partition=cp3 --qos=cp3 --wckey=cms %s\n'      % Path_Shell) 
+        cmd_file.write('sbatch --partition=Def,cp3 --qos=normal --wckey=cms %s\n'      % Path_Shell) 
```

Comment:
@clacaputo 
FYI: this has been advertised by Tier2 comp/ops in order to profit of empty slots in other partitions to minimize the time in which jobs stay pending.